packages/exchange/src/exchange/providers/ollama.py

```diff
@@ -12,18 +12,18 @@
 class OllamaProvider(OpenAiProvider):
     """Provides chat completions for models hosted by Ollama"""
 
-    __doc__ += f"""
-
-Here's an example profile configuration to try:
-
-    ollama:
-      provider: ollama
-      processor: {OLLAMA_MODEL}
-      accelerator: {OLLAMA_MODEL}
-      moderator: passive
-      toolkits:
-      - name: developer
-        requires: {{}}
+    __doc__ += f"""Here's an example profile configuration to try:
```

Comment:
need to remove the "f" here, its causing an error when it tries to interpret the {} below as a formatter